test/integration/component/test_volumes.py

```diff
@@ -603,7 +603,144 @@ def test_01_volume_attach_detach(self):
                 "Check the state of VM"
             )
         except Exception as e:
-            self.fail("Exception occuered: %s" % e)
+            self.fail("Exception occurred: %s" % e)
+        return
+    
+    @attr(tags=["advanced", "advancedns"])
```

Comment:
@dmabry Since the test can run on simulator, please add required_hardware="false" to the tags.